### PR TITLE
Limus class ship pr V2.0

### PR DESCRIPTION
**Pull request by Ketrai**
Continuation of #1173

## About The Pull Request

limus

Xenobiology focused vessel. Alternative to the Metis that leans much more heavily into xenobiology.

- Fully equipped xenobiology for two people.
- Chemlab for making mutagens to change species. (Can't be used for medical, has its own board)
- Janitoral closet to clean up the inevitable mess xenobiology/chemistry will make.
- Trash disposal shute to help get rid of the immense amount of trash xenobiology makes.
- Fooonky shaped ship
- Minor botany requiring some setup for people that don't want to handle slimes right away.
- Incomplete R&D, needs help from another ship to get it going.
- Secrets!

## Why It's Good For The Game

This is mainly meant as an alternative to the metis that leans more into xenobiology. Metis is still a far better research ship in comparison. Xenobiology takes a really long time to complete/get going. (1.-1.5 hours). And the Metis set up time is around 20 minutes to 45 mins even with multiple people. This cuts down the amount of time you need to get going with xenobiology so you can actually enjoy using the extracts more, or trading with others. (At least on short rounds that don't hit 4 hours.)

Metis also only really has proper space for one xenobiologist, so now more people in the same round can play one.

PROS :

- Xenobiology.
- Xenobio console boards.
- Better parts to get going with xenobiology properly.
- Unique chemlab that makes it a bit easier to make mutagens with xenobiology slime mutagens.
- Dedicated xenobiology space means the experience is a lot less scuffed

CONS :

- No autolathe, R&D, cargo, you basically only get xenobio.
- Bad power / engines, demands early mining.
- No materials to build consoles out of.
- No medicine for clone damage. If you get slime glomped, and you're not an IPC, go eat rocks. Alternatively cured with purple regenerative cores, a relatively early xenobio boon.
- Limited medical equipment in general, mostly ghetto tools save for a saw and a scalpel.

NOTE : I know xenobiology consoles are buggy right now, as is anything using cameras when the ship moves. The only real fix for this is using a screwdriver to reassemble them. Despite this, I don't feel like it's a good reason to deny players a slime management console. Without it, xenobio is really slow, and kinda tedious. Easily adding 30 minutes to an hour to the role. You still have the option to manually wrangle slimes, but it's really not that fun after 1-2 rounds, unless it's been a while.

## Changelog
:cl:
add: Adds the Limus, a xenobiology focused ship!
/:cl:

### CODE REVIEW

**Comment by Ketrai** [comment]
I didn't add a gun, I was told by @SweetBlueSylveon this isn't needed anymore on discord.

**Comment by Valorium-RB** [comment]
Reminder that ships should have a restroom by decree of one of our maptainers.

**Comment by Ketrai** [comment]
> Reminder that ships should have a restroom by decree of one of our maptainers.

Done.

To preserve a sense of symmetry, I've added a lil medbay to the left side. This includes a blast door so there's at the very least two ways in and out of the ship now. And this allows to pull people in for emergency treatment. Which is nice.

File: _maps/shuttles/shiptest/limus.dmm [hunk 1]
```diff
@@ -0,0 +1,5218 @@
...
+/area/ship/science)
+"fM" = (
+/obj/machinery/light,
+/obj/item/stock_parts/manipulator,
+/obj/item/stock_parts/micro_laser,
+/obj/item/stock_parts/scanning_module,
+/obj/structure/table/reinforced,
+/obj/item/reagent_containers/food/snacks/spacetwinkie{
+	desc = "Said to have been carried around by the very founder of Donk Co.";
+	name = "Blessed Twikie"
```
Comment (Apogee-dev) [inline _maps/shuttles/shiptest/limus.dmm]: name = "Blessed Twinkie"

**Comment by Apogee-dev** [comment]
Remember to give the airlocks custom names. Where possible, make sure the airlock's name string is identical (including case) to the name of the area the airlock is placed in, so that auto-naming works correctly (that is, so the airlocks get prefixed with the ship's name).

File: _maps/shuttles/shiptest/limus.dmm (hunk 1 above)
Comment (Ketrai) [reply _maps/shuttles/shiptest/limus.dmm]: fixed

**Comment by Ketrai** [comment]
> Remember to give the airlocks custom names. Where possible, make sure the airlock's name string is identical (including case) to the name of the area the airlock is placed in, so that auto-naming works correctly (that is, so the airlocks get prefixed with the ship's name).

Should be fixed

**Comment by Ketrai** [comment]
Seems people are still hesitant over merging a xenobio focused ship. (Based from conversation with Apogee) But I feel xenobio as is, is still fun enough as is to warrant giving players comfy access to for now. I'm happy to properly rework xenobio later in the year, but for now it would be a shame to see all the work put into this ship wasted. If need be I can give the ship a different focus,  or partially replace xenobio with something else. But I'd like to be told to do so outright.

**Comment by PrefabQuasar** [comment]
we apologize for not giving an official stance on this ship sooner, we are still developing the official mapping standards and neglected this PR in the process, since it stands in an unfortunate grey area, its painful to see so much effort go to waste, and we would be open to a remap or redesign of this map, but in its current state, it still carries much of the "flying workplace" design that we are trying to avoid, with the other portions seeming to be an afterthought, along with the fact that a purely xenobio-focused ship simply doesnt mesh well with the shiptest gameplay loop, you're more than welcome to draft this PR or remake the map. the fact that this has gone so far without an official decision is our fault, and we are sorry.

**Comment by Noobetski** [comment]
should have made it run on the charged yellow cores PCU, as god intended

**Pull request closed without merging**